### DISCUSSION TITLE
Hackday Introductions

### DISCUSSION THREAD

**Discussion by ateucher** [earthaccess-dev/earthaccess#813] [Hack days]
We are using this discussion thread to hold short bios / introductions for earthaccess hackday participants. This will help people who are joining for the first time know who is on the call, and allows us to create a welcoming space for newcomers while optimizing our time together. If you feel comfortable, please tell us a little bit about yourself in a comment below!

**Comment by itcarroll** [comment]
Ian Carroll here to bring Ocean Biology DAAC user needs perspectives to the contributors circle, I am at NASA Goddard Space Flight Center in the Ocean Ecology Lab. My own research is about algorithms for phytoplankton biodiversity that will use the PACE Mission's hyperspectral Ocean Color Instrument.

**Comment by DeanHenze** [comment]
Hi all, Dean Henze here with PO.DAAC. Very interested in offering cloud-optimized data sets at PO.DAAC and making it easy for users to access. Topics I'd like to explore

- Top-level function in earthaccess for accessing virtualized data sets in DAACs. 
- Exploring a virtual store, where DAACs would store this, and where in CMR such info would be stored.
- Figuring out how to approach the chunking / rechuncking challenge. How useful is a virtual data set if you cannot easily rechunk it for the needs of your specific analysis?

**Comment by ebolch** [comment]
Hi all, I am Erik Bolch, a Remote Sensing Scientist at LP DAAC. I create python Scripts and Notebooks to help users work with LP hosted datasets as well as testing and prototyping workflows for the AppEEARS service. My specialty is working with imaging spectroscopy/hyperspectral data.  I'm here to learn more about earthaccess and how to contribute.

**Comment by andypbarrett** [comment]
Hi, I'm Andy Barrett.  I work at the National Snow and Ice Data Center (NSIDC), Boulder, CO, but work remotely from Taos, NM.  I split my time between between research and supporting data use and education as part of the NSIDC Distributed Active Archive Center (DAAC).  Originally my background is in the hydrology of mountain glaciers but much of my current research focuses on Arctic sea ice and Arctic hydroclimatology.  My work for the NSIDC DAAC data use and education involves creating tutorials and other educational content to help users use NASA data more efficiently.

My work on `earthaccess` largely focuses on improving the documentation.

**Comment by nikki-t** [comment]
Hi everyone, my name is Nikki Tebaldi and I am a software engineer at the Physical Oceanography Distributed Active Archive Center (PO.DAAC). I work to migrate existing data processing operations to the cloud and to build cloud-native workflows. I have been interested in defining a plugin interface to support the use of data services in `earthaccess`.

**Comment by edshred2000** [comment]
HI all, Ed Armstrong from the PO.DAAC.  I am Project Data Engineer for the PO.DAAC and lead a task with the same name.  Super interested in the emerging cloud optimization strategies with earthaccess() and also VirtualiZarr and kerchunk.    I also support the Nisar on-demand cloud processing system and helped cloud optimize their HDF5 file format using metadata aggregation.  I hope to start a project soon to migrate a PO.DAAC L4 SST processing system to the cloud.

**Comment by mfisher87** [comment]
:wave: I'm Matt, a research software engineer at NSIDC, at least for a couple more weeks :) I love writing Python, I write a bit of TypeScript too I guess ;), and my favorite part of software development is the community aspect. It's just more fun to develop stuff together. And the outcomes are better. I also recently discovered, through designing and administering [a workshop for researchers](https://qgreenland-workshop-2023-researcher.github.io/), that I love to teach! I'm now a certified [Carpentries](https://carpentries.org) instructor. I especially like to teach about Git at any level :)

I will shortly be moving to a new role at Schmidt Center for Data Science and Environment (DSE) at UC Berkeley. I'll still be around to participate in and help facilitate _earthaccess_ hack days :) And I'll be starting up a new series of hack events for the [GeoJupyter](https://github.com/geojupyter/) project. Follow us on GitHub for updates!
